schemas/dab.draft.schema.json

```diff
@@ -874,6 +922,16 @@
       }
     }
   },
+  "if": {
```

Comment:
do we have a test to verify schema validation errors if endpoint is not provided?